spec/puppet-server-lib/puppet/jvm/ast_compiler_spec.rb

```diff
@@ -135,5 +149,41 @@ def find_notify(catalog, title: nil)
         end
       end
     end
+
+    context 'when running a project plan' do
+      let(:boltlib_path) { [] }
+      context 'single threaded' do
+        it 'returns puppet settings back to normal' do
```

Comment:
Is this test still necessary? It seems like not...I think we are not munging the settings any more?

Reply:
deleted.